src/angular.headroom.js

```diff
@@ -25,6 +25,29 @@
         $scope.$on('$destroy', function(){
           headroom.destroy();
         });
+	
+        //Setup watchers to update headroom object properties on attibute change
+        $scope.$watch('tolerance', function (newTolerance) {
+          if (newTolerance) {
+            headroom.tolerance = newTolerance;
+          }
+        });
+        $scope.$watch('offset', function (newOffset) {
+          if (newOffset) {
+            headroom.offset = newOffset;
+          }
+        });
+        $scope.$watch('classes', function (newClasses) {
+          if (newClasses) {
+            headroom.classes = newClasses;
+          }
+        });
+        $scope.$watch('scroller', function (newScroller) {
+          if (newScroller) {
+            headroom.scroller = newScroller;
```

Comment:
`newScroller` is just a string (mentioned in conversation comments already).

`headroom.scroller = document.querySelector(newScroller);`